netbox/extras/api/views.py

```diff
@@ -243,7 +243,7 @@ def list(self, request):
             'request': request,
         })
 
-        return Response(serializer.data)
+        return Response({'count': len(report_list), 'results': serializer.data})
```

Comment:
Should `display` also include the module? If not, we could see duplicate report/script names.

Reply:
Check now if you find the format correct or should we use a different one?